core/src/main/java/com/scalar/db/storage/jdbc/JdbcAdmin.java

```diff
@@ -429,6 +441,62 @@ public TableMetadata getTableMetadata(String namespace, String table) throws Exe
     return builder.build();
   }
 
+  @Override
+  public TableMetadata getImportTableMetadata(String namespace, String table)
+      throws ExecutionException {
+    TableMetadata.Builder builder = TableMetadata.newBuilder();
+    boolean primaryKeyExists = false;
+
+    if (!rdbEngine.isImportable()) {
+      throw new UnsupportedOperationException(
+          "Importing table is not allowed in this storage: " + rdbEngine);
```

Comment:
`RdbEngineStrategy` doesn't override `toString()`, so maybe `rdbEngine.getClass().getName()` is more readable:
```suggestion
          "Importing table is not allowed in this storage: " + rdbEngine.getClass().getName());
```

Reply:
Fixed in 0d3a2f8.